### PR TITLE
post: visual showcase of neovim capabilities

### PR DESCRIPTION
**Pull request by teto**
I would like to showcase the folds/fillchars with unicode too.

It would be nice if we could issue more blog posts to communicate about 0.5, for instance one introduction about treesitter, one about lsp, and some visual showcase. This is the latter, don't hesitate to propose screenshots that are cool and exhibit neovim features. It may start as a blogpost and/or end up on the website permanently.

I have used your pictures, let me know if you disagree @smolck @akiyosi  @gleipnir @sunjon

### CODE REVIEW

File: _posts/2020-09-14-screenshots.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+Neovim: the beauty and the beast
+----------
+
+TUI = beast
+
+then move on to 
+
+![Courtesy of sunjon](../images/showcase/telescope_helptags.png)
+![Courtesy of smolck veonim](../images/showcase/veonim_nyancat.png)
```
Comment (clason) [inline _posts/2020-09-14-screenshots.md:39]: Please make sure to credit https://github.com/veonim/veonim for this picture ("official" pic: https://github.com/veonim/veonim/blob/release/docs/nyan.png)

**Comment by teto** [comment]
actually I think it should be a "screenshot" page on the website. It's always nice to have some screenshots, to know what's possible.

**Comment by teto** [comment]
the goal of the PR changed a bit, this creates a screenshot section. I admit it's not pretty but I've reached the limit of my capabilities. Maybe a designer will have a heartattack seeing it and improve on it (I wish :D ) : 2021-04-09_22-25-screenshots

**Comment by teto** [comment]
@glepnir if you want to share a screenshot, you can do it here or on gitter (sry if I missed it).

**Comment by smolck** [comment]
@teto Not sure if you’d want to use any of these, but just in case note that there are some screenshots in [uivonim’s wiki](https://github.com/smolck/uivonim/wiki/Builtin-LSP) which showcase the external LSP GUI elements.

No worries though if you don’t want to use any of them since you already have an uivonim/veonim screenshot, just thought I’d mention them since I don’t know of any other GUI that has those sort of ext elements (except for GNvim’s external hover feature showcased at the bottom of https://github.com/vhakulinen/gnvim/wiki).

**Comment by teto** [comment]
@teto I've added one additional screenshot for uivonim. I tried adding titles under the images but it was not pretty. So if you are still feel like improving it, be my guest xD

@vhakulinen would it be ok to use a screenshot from https://github.com/vhakulinen/gnvim/wiki ?

We can replace screenshots if better ones appear, anyhow this PR strictly improves the existing (IMO) so I am going to merge soon (before 0.5).

**Comment by vhakulinen** [comment]
Sure, you can use any screenshot found from gnvim's wiki.

**Comment by teto** [comment]
Thanks ! Let us know if at some point you want to replace it with a better one.

@smolck can I let you add a gnvim screenshot as well ? I think screenshot3 is nice but that's up to debate (trying to show a different aspect of neovim with every screenshot).

**Comment by smolck** [comment]
So I made some fairly small formatting/wording/etc. changes which I'll just paste here since I can't commit to this branch:
<details>

<summary>Diff</summary>

```diff
diff --git a/screenshots.html b/screenshots.html
index f2646e6..73c31d2 100644
--- a/screenshots.html
+++ b/screenshots.html
@@ -8,16 +8,40 @@ title: Screenshots
 <div class="container golden-grid">
   <div>
     <h2>Disclaimer</h2>
-	The following screenshots are taken possibly from an unreleased version of neovim, to showcase what is possible with a few plugins and lines of configuration: they are not representative of an out of the box experience.
+	The following screenshots are taken from possibly unreleased versions of
+    Neovim, to showcase what is possible with a few plugins and lines of configuration: they are not representative of an out of the box experience.
+
     <h2>Terminal screenshots</h2>
-	This section shows screenshots from the terminal,
-	<img src="/images/showcase/telescope_helptags.png" alt="TUI with telescope (courtesy of sunjeon)" width="80%">
-
-	<img src="/images/showcase/floating-win.png" alt="courtesy of gleipnir" width="80%"/>
-    <h2>Graphical interfaces screenshots</h2>
-	Graphical interfaces are not packaged with neovim. This section showcases only a handful of them, see neovim's wiki <a href="https://github.com/neovim/neovim/wiki/Related-projects#gui">for a detailed list</a>.
-	<img src="/images/showcase/uiveonim-references.png" alt="uivonim (courtesy of smolck)" width="80%"/>
-	<img src="/images/showcase/veonim_nyancat.png" alt="uivonim (courtesy of smolck)" width="80%"/>
-	<img src="/images/showcase/goneovim.png" alt="goneovim (courtesy of akyosi)" width="80%"/>
+    <ul>
+      <li>
+        <h5>Custom Start Screen and <a href="https://github.com/nvim-telescope/telescope.nvim">Telescope</a></h5>
+        <img src="/images/showcase/telescope_helptags.png" alt="TUI with telescope (courtesy of sunjon)" width="80%">
+      </li>
+
+      <li>
+        <h5>LSP Search with <a href="https://github.com/glepnir/lspsaga.nvim#async-lsp-finder">Lspsaga</a></h5>
+        <img src="/images/showcase/floating-win.png" alt="courtesy of gleipnir" width="80%"/>
+      </li>
+    </ul>
+
+    <h2>GUI Screenshots</h2>
+	GUIs are not packaged with Neovim. This section showcases only a handful of them, see <a href="https://github.com/neovim/neovim/wiki/Related-projects#gui">Neovim's wiki</a> for a list of many of the available GUIs.
+
+    <ul>
+      <li>
+        <h5>LSP References UI in <a href="https://github.com/smolck/uivonim">Uivonim</a></h5>
+        <img src="/images/showcase/uiveonim-references.png" alt="uivonim (courtesy of smolck)" width="80%"/>
+      </li>
+
+      <li>
+        <h5>Nyancat in <a href="https://github.com/smolck/uivonim">Uivonim</a></h5>
+        <img src="/images/showcase/veonim_nyancat.png" alt="uivonim (courtesy of smolck)" width="80%"/>
+      </li>
+
+      <li>
+        <h5><a href="https://github.com/akiyosi/goneovim">Goneovim</a></h5>
+	    <img src="/images/showcase/goneovim.png" alt="goneovim (courtesy of akyosi)" width="70%"/>
+      </li>
+    </ul>
   </div>
 </div>
```
</details>

Not 100% sure about the text I came up with for the headers & the re-wording I did, lmk what you think.

Also here are better screenshots for uivonim you may want to use instead of the current ones, the first is one of nyancat that isn't a weird aspect ratio & that doesn't have my wallpaper in the background, and the second is one of the references UI but w/a smaller window, different colorscheme, & different PL. Feel free to keep the ones you have, I just figure these are better.
<details>

<summary>Better screenshots</summary>

Screen Shot 2021-04-24 at 11 59 51 PM

Screen Shot 2021-04-25 at 12 07 14 AM

</details>

> @smolck can I let you add a gnvim screenshot as well ? I think screenshot3 is nice but that's up to debate (trying to show a different aspect of neovim with every screenshot).

@teto You mean just take a screenshot and post it here? Or . . .

**Comment by glepnir** [comment]
Aha okay I will update some new screenshots.

**Comment by teto** [comment]
I've added nvim-qt (I had not realized they had progressed so much ! they have a treeview/externalized context menu) and gnvim. Just waiting for your screen glepnir then I merge.

**Pull request merged**